Meta/CMake/FindSwiftTesting.cmake

```diff
@@ -17,6 +17,7 @@ if (SWIFT_TESTING)
         cmake_path(GET _SWIFT_TESTING_DIR PARENT_PATH _SWIFT_TESTING_TARGETLESS_DIR)
         set_target_properties(SwiftTesting::SwiftTesting PROPERTIES
           INTERFACE_COMPILE_OPTIONS "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-load-plugin-library ${_SWIFT_TESTING_TARGETLESS_DIR}/host/plugins/libTestingMacros.so>"
+          INTERFACE_LINK_OPTIONS "-load-plugin-library;${_SWIFT_TESTING_TARGETLESS_DIR}/host/plugins/libTestingMacros.so"
```

Comment:
This hack is particularly sad: Because the INTERFACE_COMPILE_OPTIONS I made for loading the plugin library for swift-testing has a COMPILE_LANGUAGE genex on it, I can't use it directly in the custom command in `_swift_generate_cxx_header`. Because an `add_custom_command` is not a "compile command", CMake balks at it. So I went with a fragile hack of putting the settings in INTERFACE_LINK_OPTIONS.

I'm not 100% sure, but I think that this will crash and burn if ladybird adds its own macro library due to option deduplication.